charts/ark-solr/Readme.md

```diff
@@ -0,0 +1,11 @@
+The following table shows the configuration options for the Solr helm chart:
+| Parameter                                     | Description                           | Default Value                                                       |
+| --------------------------------------------- | ------------------------------------- | --------------------------------------------------------------------- |
+| `port`                                        | The port that Solr will listen on | `8983`                                                                |
+| `service.port`                                        | The port that Solr will listen on | `8983`                                                                |
+| `image.repository`                                        |  |  check the values.yaml for more information                                                           |
+| `image.tag`| | check the values.yaml more information|
+| `nameOverride`| name of the k8s workload ad chart name | `ark-solr` |
+|`service.type`| service type of k8s workload for internal communication | `ClusterIP`|
+|`persistence.storageClass` | type of storage used | `gp2`|
+|`persistence.size` | persistent volume size | `8`|
```

Comment:
@naveen-armedia If you want to use a table in a .md file, please ensure the columns are aligned so the table is readable in a text editor using a fixed-size font.

Reply:
@fabricetriboix  In preview  .md file looks to be fine. I can make each column with equal space in .md file but preview would result the same.

Reply:
@naveen-armedia Somebody with a basic text editor must be able to read a .md. Please use notepad or somesuch to understand what I mean. Thanks.

Reply:
This file wasn't be modified yet, as far as I can see...

Reply:
@naveen-armedia That still wasn't good. I reformated the file, I hope that will be OK with you.